extensions/wikia/Helios/User.class.php

```diff
@@ -198,6 +198,78 @@ public static function onUserCheckPassword( $id, $username, $hash, $password, &$
 		return true;
 	}
 
+    /**
+     * Called in ExternalUser_Wikia registers a user.
+     *
+     * @param string $username string of the user name
+     * @param string $password string of the plaintext password the user entered
+     * @param string $email string of the user email
+     *
+     * @return boolean true on success, false otherwise
+     */
+    public static function register( $username, $password, $email )
+    {
+        $logger = \Wikia\Logger\WikiaLogger::instance();
+        $logger->info( 'HELIOS_REGISTRATION', [ 'method' => __METHOD__ ] );
+
+        global $wgHeliosBaseUri, $wgHeliosClientId, $wgHeliosClientSecret;
+        $helios = new Client( $wgHeliosBaseUri, $wgHeliosClientId, $wgHeliosClientSecret );
```

Comment:
Instantiation of the Client class happens at least twice in this file and may be a good candidate for method extraction.